core/src/main/scala/laserdisc/protocol/RESPFrame.scala

```diff
@@ -41,12 +39,10 @@ private[laserdisc] final case class CompleteFrame(bits: BitVector) extends NonEm
 private[laserdisc] final case class MoreThanOneFrame(private[laserdisc] val complete: Vector[CompleteFrame], remainder: BitVector)
     extends NonEmptyRESPFrame
 private[laserdisc] final case class IncompleteFrame(partial: BitVector, bitsToComplete: Long) extends NonEmptyRESPFrame {
-  override def append(bytes: ByteBuffer): Exception | NonEmptyRESPFrame = {
-    val newBits = BitVector.view(bytes)
+  override def append(bits: BitVector): Exception | NonEmptyRESPFrame =
```

Comment:
here's the only real change to production code, and it's more a clean up of unneeded conversions than a performance improvement.